### ISSUE TITLE
Add ability to set "hover" or "click" to display navigation menu drop down

### ISSUE THREAD

**Issue by brianyerkes** [elementor/elementor#9638]
## Prerequisites

- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
Allow for the ability to change the hover feature for navigation menus to a click function instead. Click functions are much more user friendly for displaying a sub menu as the user doesn't need to carefully keep their pointer positioned within the sub menu area to keep it visible/available.

**Describe the solution you'd like**
An option in the nav menu element to set "hover" or "click" options for displaying a sub menu on a nav menu link that has a sub menu.

**Describe alternatives you've considered**
Custom coding the header file.

**Additional context**

**Comment by deep-ops** [comment]
Yes, please, I also want this feature and my clients do as well.

**Comment by Mike-Benoit** [comment]
Yes, on-hover menus cause major confusion for users, even at the top-level. 

We see almost all of our "rage clicks" on the menu, because the user sees the menu, and they are trained to click on menu items, as almost all software works that way. They move the mouse up to the menu, there is a ~.25s delay before it opens, which is really close to the time it takes a human to click, so the menu opens just as the user clicks, which closes the menu. So now they need to click **again** to open it. Very frustrating. 

Additionally, the mouse cursor shows the text selection cursor when hovering over the top level of the menu. Sub-menus show the hand. Ideally it should show the hand cursor over any part of the menu.

**Comment by Mike-Benoit** [comment]
Since this issue is about 4 years old, and unlikely to ever be addressed, below is a workaround to do it outside of Elementor. Replace "menu-1-157a6843" with your own elementor menu ID of course. I inserted it into the bottom of the page using the free "WPCode Lite" plugin.

```
<script>
jQuery(function() {
    setTimeout(function() {
        jQuery('#menu-1-157a6843').unbind('mouseenter');
        jQuery('#menu-1-157a6843 > li > a').css('cursor', 'pointer');
        jQuery('#menu-1-157a6843 > li > a').css({
            '-webkit-user-select': 'none', // Safari and Chrome
            '-moz-user-select': 'none',    // Firefox
            '-ms-user-select': 'none',     // IE 10+
            'user-select': 'none'          // Future-proofing for planned standards
        });		
    }, 250); // Delay in milliseconds, adjust as needed
});
</script>
```

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with the new Menu widget in Elementor Pro v3.12.0! 🥳 

You can now decide if the sub-menu items are opened on hover or on click in the new mega Menu widget.

Learn more here: https://elementor.com/help/the-menu-widget/

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
